packages/shared/src/modules3rdParty/cross-crypto/index.native.js

```
@@ -1,27 +1,31 @@
+/* eslint-disable no-undef */
 // react-native-crypto
 //      react-native-quick-crypto
 // react-native-get-random-values
 //      (react-native-crypto depend on) react-native-randombytes (deprecated)
 console.log('================ cross-crypto (native)');
 
-if (global.crypto && global.crypto.getRandomValues) {
-  delete global.crypto.getRandomValues;
+// delete globalThis.crypto.getRandomValues first
+// make sure react-native-get-random-values can be shimmed
+if (globalThis.crypto && globalThis.crypto.getRandomValues) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)